source/braille.py

```diff
@@ -1677,7 +1690,16 @@ def handleGainFocus(self, obj):
 
 	def _doNewObject(self, regions):
 		self.mainBuffer.clear()
+		focusToHardLeftSet = False
 		for region in regions:
+			if self.tether == self.TETHER_FOCUS and config.conf["braille"]["focusContextPresentation"]=='hybrid':
+				if region.focusToHardLeft:
+					focusToHardLeftSet = True
+				elif not focusToHardLeftSet and getattr(region, "_focusAncestorIndex", None) is None:

```

Comment:
I'm afraid your incorrect here. This code applies to the first region that is returned by getFocusRegions, and this doesn't necessarily have to be regions-1 if I'm correct.